docs: Add CLI docs for 0.16.1 and 0.17.0 (#1300)
keptn/keptn.github.io: content/docs/0.17.x/reference/cli/commands/keptn_trigger_sequence.md

---
date: "2022-07-11T09:29:13+02:00"
title: "keptn trigger sequence"
slug: keptn_trigger_sequence
---
## keptn trigger sequence

Triggers the execution of any sequence in a project

### Synopsis

> Triggers the execution of any sequence in a project with an arbitrary name.

Reviewer: ```suggestion
Triggers the execution of the specified sequence in a project.
```
